atoma-bin/atoma_node.rs

```diff
@@ -212,6 +212,8 @@ async fn main() -> Result<()> {
         event = "p2p_node_spawn",
         "Spawning Atoma's p2p node service"
     );
+    validate_country_code(&config.p2p.country)?;
```

Comment:
you can use the validator crate to add this to the config directly, just slightly change the result of the function to ValidationError. We should add also the public url validtor directly to the config. That way you can just look at the config struct and see which validators do we have in place.

```
    #[validate(custom(function = "validate_country_code"))]
    pub country: String,
 ```